tests/snippets/test_frame.py

```diff
@@ -556,3 +556,10 @@ def test_types_to_latex() -> None:
     df.to_latex(position='some')
     # caption param was extended to accept tuple in 1.2.0 https://pandas.pydata.org/docs/whatsnew/v1.2.0.html
     df.to_latex(caption=("cap1", "cap2"))
+
+
+def test_explode() -> None:
```

Comment:
The naming convention here should be `test_types_x`, so if you could please change to `test_types_explode`, that would be great!

Reply:
Fixed! Thanks for the feedback!